app/inbox/inbox.tsx

```diff
@@ -41,6 +42,20 @@ function Inbox() {
     }
   }, [currentWorkspace?.id]);
 
+  // Filter ticket based on activeTab
+  const filteredTicketList = ticketList?.filter((ticket) => {
```

Comment:
Will not do like this in useEffect we will set filtered list and then use it
we call that function when we change tab and it will reset filtered list

Reply:
Done